pysat/_meta.py

```diff
@@ -1270,9 +1271,11 @@ class MetaLabels(object):
     desc : tuple
         Description label name and value type (default=('desc', str))
     min_val : tuple
-        Minimum value label name and value type (default=('value_min', float))
+        Minimum value label name and value type (default=('value_min',
+        float))
     max_val : tuple
-        Maximum value label name and value type (default=('value_max', float))
+        Maximum value label name and value type (default=('value_max',
+        float))
```

Comment:
Again, it looks like there's no reason for this change.

Reply:
See above but fixed.